### PR TITLE
Fix #1147 - Adds README for new community fund process to CPC repository

### LINKED ISSUE #1147: Revamping the travel fund
As traveling is picking up again we're coming close to max out our travel fund this year.

Additionally, the fairly ad hoc processes we had in place aren't scaling up that well. They also have [privacy issues](https://github.com/openjs-foundation/cross-project-council/issues/397) that we are now committed to fix.

As a result we're going to revamp the travel fund with the following draft goals and requirements in mind (this is an opinionated summary of two different [working](https://github.com/openjs-foundation/cross-project-council/issues/397#issuecomment-1664719695) [sessions](https://docs.google.com/document/d/13Ocz4VMn-31CT8ZaqUZ0U_fLeMWJqRs2mwStmx_EChA/edit) on the topic):

1. **provide financial support for active independent contributors** to participate in events that benefit OpenJSF projects, working groups, or collab spaces.
2. **support diversity and inclusion**.
3. be publicly **transparent** about how those funds are used
    - be able to share anonymized data publicly, for example:
        - funds requested vs. fund accepted
        - how much funds are distributed by project/WG/collab space
        - how much  funds are distributed by event
        - how much funds are distributed to independent contributors vs. employed contributors
        - how much funds are distributed to contributors who self-report as member of an underrepresented minority
    - have a private papertrail of what decisions were made, by whom, and their rationale
4. be mindful of contributor's **privacy**
    - make decisions in private
    - do not share names or location of funded individuals without their agreement
    - agreement for information to be shared publicly cannot have impact on eligibility
5. be **responsive**
    - acknowledge receipt of request immediately
    - approve or reject requests quickly (ideally within 2 business days or at a given date if we manage requests in batches for specific events)
    - if fund is rejected, inform requester as to why, with escalation path (CPC meeting?)
    - if request needs escalation, inform requester right away plus explain why
6. be **efficient**
    - delegate process and most decision-making to foundation staff with the ability to escalate back to the CPC or project maintainers when necessary
    - have clear guidelines for how to approve requests
    - request relevant information from requesters to identify
        - which projects they're contributing to,
        - if they're doing so in a personal capacity or as part of their job
     - escalation path to maintainers to approve active contribution when in doubt
         - have a list of project maintainers
         - have clear guidelines for maintainers on how to assess eligibility
     - escalation path to CPC for any contentious issue
     - use a solution that CPC members and project maintainers can be onboarded to quickly
     
## Workstreams / Areas Of Improvement
- New user flow, from a contributor perspective how do they request travel funding? (Google form?)
- CPC approval flow, how does the CPC intake and ultimately approve funding requests? (list of main events we're expecting to support, request batching, announcing deadline by which requests have to be received, CPF-style)
- Budget tracking, reporting and yearly budget setting (potentially dividing up among groups)
- Transparency reports
- Clarify purpose of Fund (goal setting and related policy)

```[tasklist]
### Tasks
- [ ] https://github.com/openjs-foundation/cross-project-council/issues/1154
- [ ] https://github.com/openjs-foundation/cross-project-council/issues/1164
- [ ] https://github.com/openjs-foundation/cross-project-council/issues/1165
- [ ] https://github.com/openjs-foundation/cross-project-council/issues/1166
- [ ] https://github.com/openjs-foundation/cross-project-council/issues/1167
```

### PR DESCRIPTION
**Pull request by bensternthal**
This PR includes many updates to the community fund intake, review, and approval process. Highlights of this effort include

- Moving these docs into this repo, we will archive https://github.com/openjs-foundation/community-fund/ once merged
- Adding the form for intake
- Better definition of what the fund will cover
- Adding the DEI fund allocation
- Adding a new cadence and process for CPC reviews
- And more

Much thanks to @joesepi @PaulaPaul @tobie @mcollina @rginn and all the other folks in the CPC that contributed to this work.

### CODE REVIEW

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (ljharb) [inline community-fund/COMMUNITY_FUND_POLICY.md:11]:
```suggestion
The fund can cover the costs, in whole or in part, of a participant's hotel, airfare, and some event registrations.
Applicants should explore employer-sponsored funding options as their first source of financial support.
Additionally, 25% of the community fund is allocated specifically to support diverse participants.
Funding decisions are determined by a combination of need and potential impact.
```

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (ljharb) [inline community-fund/COMMUNITY_FUND_POLICY.md:21]:
```suggestion
NOTE: Funds may not be used for miscellaneous travel expenses including food, visa costs, non-airport transportation, baggage fees, etc.
Generally, registration can be reimbursed if the attendee is speaking on behalf of OpenJS, the CPC, or OpenJS-hosted projects.
Funding is not transferable to another person, or any other event. 
```

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (ljharb) [inline community-fund/COMMUNITY_FUND_POLICY.md:29]:
```suggestion
Submit your travel fund request to the Cross Project Council by completing [this application](https://forms.gle/QDt3iqoXXB5Ycovz8).
Please see [Deadlines and Timelines](#deadlines-and-timelines) for information about cut-off dates. 
```

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (ljharb) [inline community-fund/COMMUNITY_FUND_POLICY.md:71]: can "OpenJS Foundation Slack" be a link to a channel?

**Review by ljharb — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (tobie) [inline community-fund/COMMUNITY_FUND_POLICY.md:73]:
```suggestion
- Approval will be made through the [consensus seeking process](https://github.com/openjs-foundation/cross-project-council/blob/main/governance/GOVERNANCE.md#consensus-seeking-process).
```
There's no reasons, imho, to introduce a new decision-making mechanism here.

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (tobie) [inline community-fund/COMMUNITY_FUND_POLICY.md:50]:
```suggestion
Please note that expenses without receipts cannot be paid.
```
Nit.

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (tobie) [inline community-fund/COMMUNITY_FUND_POLICY.md:47]: There are three docs in that folder. Could we maybe link directly to the two forms?

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (tobie) [inline community-fund/COMMUNITY_FUND_POLICY.md:21]:
```suggestion
* Accommodations (dates of the event plus one night prior are eligible for reimbursement)
```

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (tobie) [inline community-fund/COMMUNITY_FUND_POLICY.md:20]:
```suggestion
* Airfare tickets (in economy class)
```
"Coach" is very North American.

**Review by tobie — APPROVED** [review]

**Comment by PaulaPaul** [comment]
Hi all - sorry I was not able to join the CPC call yesterday, it looks like there was a lot of progress on this!

@bensternthal is there an open process to track this goal/statement? *Additionally, 25% of the community fund is allocated specifically to support diverse participants.*?  I like this idea and maybe it needs a little more definition.

Also, should this read 'Additionally, a minimum of 25% will be allocated to support diverse participants' (meaning that there is not a cap?)

If for any reason (in any given quarter?) if we are struggling to achieve the 25% goal would we be able to allocate those funds to other mechanisms that will help us build a diverse speaker/contributor community?

If the Travel Fund is the only mechanism in the Foundation that tries to build a diverse community of contributors, that does put a lot of weight on the Travel Fund; is it worth considering other mechanisms or considering this as a part of a larger umbrella of activities that can build a diverse community of contributors?

No intentions to ask for a block on any of these questions, but hoping to discuss at next week's call - we always expected that the Travel Fund would evolve as we learn so this seems like a good time to 'ship it' - thanks everyone!

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (mhdawson) [reply community-fund/COMMUNITY_FUND_POLICY.md:73]: I looked at what we have in the Node.js project and it has the link to the wiki page but nothing else. My key point in the in the recent working group meeting is that the "consensus" must represent that of the CPC, not just the subset of CPC members who could make a particular meeting. That requires that people not in attendance have the opportunity to object before the decision is considered final. I think we might want to update the description in https://github.com/openjs-foundation/cross-project-council/blob/main/governance/GOVERNANCE.md#consensus-seeking-process to reflect that as well. For example, if two people show up at a CPC meeting they are not able to make a decision on behalf of all of the CPC.  Further even if 51% of the people show up at a meeting the other 49% of the members should still have an opportunity to object before the decision is considered final.

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (aduh95) [inline community-fund/COMMUNITY_FUND_POLICY.md:11]: Air travel is not the only transportation method
```suggestion
The fund can cover the costs, in whole or in part, of a participant's hotel, transportation, and some event registrations.
```

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (aduh95) [inline community-fund/COMMUNITY_FUND_POLICY.md:22]: What about folks who skip the airport (because e.g. they live close enough)?
```suggestion
* Ground transportation to/from the airport/event
```

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (eemeli) [inline community-fund/COMMUNITY_FUND_POLICY.md:24]: If/as we're interested in increasing diversity, why are visa costs specifically not allowed? This is a cost almost solely related to potential applicants from underrepresented countries.

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (mcollina) [reply community-fund/COMMUNITY_FUND_POLICY.md:20]: Trains should be covered too.

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (ruyadorno) [reply community-fund/COMMUNITY_FUND_POLICY.md:13]: > For example, the diversity of participants and viewpoints at a European event could be improved by funding the travel of someone from Canada. Is that potentially covered from this allocation?

Agreed that there is some ambiguity here that might make it difficult for the council to review applications. Is the diversity / underrepresentation applied to the country of destination of the event? Or is it applied from a global tech scene point of view?

Another example: A collaborator from Colombia may fit well the diversity definition from a conference organized in the global north but will every Colombian collaborator be eligible in case the conference is held in Colombia?

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (ruyadorno) [reply community-fund/COMMUNITY_FUND_POLICY.md:24]: Big +1, growing up in that situation I know very well how visa costs are a huge deterrent from international travel. Not only visas but also these miscellaneous items can be definitely prohibitive (or a lot of uncertainty) for someone travelling from a country with low purchase power.

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (ruyadorno) [reply community-fund/COMMUNITY_FUND_POLICY.md:20]: it sounds to me like "Coach airfare" is a tentative of saying "the cheapest airfare tickets only, no premium, business or any additional cost allowed" - if that's the case it may be better to lay it out in plain english for the global audience

Comment (ruyadorno) [reply community-fund/COMMUNITY_FUND_POLICY.md:20]: nvm, quickly googling it tells me it's a synonym for "Economic Class" 😊 so yeah, let's please rephrase that

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (ruyadorno) [inline community-fund/COMMUNITY_FUND_POLICY.md:60]: I'm not sure how much can be done about it but I'd like to point out that having to wait a full month for reimbursement can be a big deterrent for collaborators living in countries with a low purchase power currency that might be paying more than the equivalent of a month's of salary for an airline ticket alone.

Any improvement on this might be helpful to increasing the diversity of applicants.

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (ljharb) [reply community-fund/COMMUNITY_FUND_POLICY.md:28]: Just noticed this; is this a new restriction?

Comment (joyeecheung) [inline community-fund/COMMUNITY_FUND_POLICY.md:28]: This seems like an odd restriction, presumably if someone used the fund to go to a summit then they would not be able to request it again for standard body participation, etc. which could end up hurting what this fund is for. I think a limit based on the total amount of funds requested per year per person makes more sense, or this can be capped at a value higher than 1 (maybe 2 or 3).

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (tobie) [reply community-fund/COMMUNITY_FUND_POLICY.md:73]: @mhdawson: I agree 100% on both needing to tighten our definition of consensus and on doing it in the right place (i.e. not here). Should we open an issue for this?

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (tobie) [reply community-fund/COMMUNITY_FUND_POLICY.md:28]: Agreed. Either put a cap on the max yearly allocation per person as a % of the overall budget or a $ amount or just add a more open statement such as "Funding will be allocated fairly so that no single individual benefits excessively from it." (I'd favor the later which clarifies the intention, and leaves useful wiggle room.)

Comment (mhdawson) [reply community-fund/COMMUNITY_FUND_POLICY.md:28]: This is one of the differences from the past.

Comment (ljharb) [reply community-fund/COMMUNITY_FUND_POLICY.md:28]: https://github.com/openjs-foundation/cross-project-council/pull/1230#discussion_r1457788253 has more discussion on why this restriction may not be ideal.

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (tniessen) [inline community-fund/COMMUNITY_FUND_POLICY.md:24]: This appears to be focused entirely on transportation by air carriers. Let's at the very least explicitly cover climate-friendly alternatives, such as trains, as well.

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (WaleedAshraf) [reply community-fund/COMMUNITY_FUND_POLICY.md:24]: I did benefit from reimbursing visa cost for my first Node.js summit in Canada + getting an invitation letter from LF for visa process.

Also, I'm not sure if the person requesting, mentions the miscellaneous cost at the time of approval, and it gets approved, even then they can't use it? OR they can't request funds for miscellaneous cost? Wording should be updated.

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (bensternthal) [reply community-fund/COMMUNITY_FUND_POLICY.md]: This item will need more discussion. I will add a placeholder for the purposes of merging this PR and create an issue specific to resolving this question.

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (bensternthal) [reply community-fund/COMMUNITY_FUND_POLICY.md:60]: Unfortunately, this is the LF policy expense reimbursement and is not something we can change.

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (bensternthal) [reply community-fund/COMMUNITY_FUND_POLICY.md:18]: Added a note to clarify this. Only pre-approved expenses are eligible for reimbursement.

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (bensternthal) [reply community-fund/COMMUNITY_FUND_POLICY.md]: Rephrased and added additional valid modes of transportation.

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (bensternthal) [reply community-fund/COMMUNITY_FUND_POLICY.md:24]: This item requires more discussion. I will create an issue to track this specific question.

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (bensternthal) [reply community-fund/COMMUNITY_FUND_POLICY.md:73]: Yes please spin this off into another issue so that we can unblock merging this pr.

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (bensternthal) [reply community-fund/COMMUNITY_FUND_POLICY.md:47]: One of those are directions. I would prefer to just link to this directory.

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (bensternthal) [reply community-fund/COMMUNITY_FUND_POLICY.md:13]: 25% is an allocation and we would not be limited to that amount, going over to support diverse candidates would be fantastic.

We were given specific legal guidance on what information we can collect regarding diversity. Additionally we were given guidance on what specific language to use. 

In this context diversity means identifying from an underrepresented group and this is open to interpretation by the submitter. Geographic location of the person is collected as part of the form and the CPC can and should factor this into their decisions.

**Review by bensternthal — COMMENTED** [review]
Integrated feedback and suggestions

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (bensternthal) [reply community-fund/COMMUNITY_FUND_POLICY.md:73]: I spun this out into it's own issue and will remove the text from this PR pending the CPC's decision.

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (bensternthal) [reply community-fund/COMMUNITY_FUND_POLICY.md]: Removed line pending discussion in https://github.com/openjs-foundation/cross-project-council/issues/1239

**Review by mhdawson — APPROVED** [review]
LGTM

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (tobie) [reply community-fund/COMMUNITY_FUND_POLICY.md:60]: I'm sorry to be annoying here, but OpenJSF is an independent organization and can have its own policy on expense reimbursement. I don't think we should block on this, but we should definitely file a separate issue and aim to address it.

**Review by joesepi — APPROVED** [review]

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (eemeli) [reply community-fund/COMMUNITY_FUND_POLICY.md:13]: Is any guidance provided elsewhere to the applicants or the CPC about how eligibility for being diverse is determined? In particular, the questions @ruyadorno raises above about what the point of view is for determining diversity and underrepresentation don't seem addressed at all.

Comment (bensternthal) [reply community-fund/COMMUNITY_FUND_POLICY.md:13]: This is my understanding, so I would love others @joesepi & @PaulaPaul to jump in if I state anything incorrectly.

We've tried to implement this in a way that the responsibility of deciding who is from an underrepresented group  does not fall to the CPC.  The form asks whether the applicant identifies as part of such a group. If they affirm, we accept their self-identification.

Comment (ovflowd) [reply community-fund/COMMUNITY_FUND_POLICY.md:13]: As far as I recall, we decided not to be explicit on what being a diverse individual entitles and let people decide if they believe they belong to underrepresented groups. It is a vote of fairness and trust between the CPC and the applicant. Of course, if the system is abused (or we have reasons to believe a candidate is abusing the system), their travel fund request will (very possibly) be rejected.

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (ovflowd) [reply community-fund/COMMUNITY_FUND_POLICY.md:47]: Can we maybe have a perma-link (shortened link) to this doc? Is this document under someone's individual account or some Foundation-shared folder?

Comment (ovflowd) [reply community-fund/COMMUNITY_FUND_POLICY.md:47]: I feel that we might actually want to upload these docs in this repository just to ensure they're always available. And yes, it'd be great if we were more explicit about what each form is.

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (ovflowd) [reply community-fund/COMMUNITY_FUND_POLICY.md:60]: @tobie is that issue link correct? It seems to link to another issue?

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (ovflowd) [inline community-fund/COMMUNITY_FUND_POLICY.md:72]: This section seems commented out. Is this intentional?

File: README.md
Comment (ovflowd) [inline README.md:234]: Is this https://github.com/openjs-foundation/community-fund repository still relevant?

**Review by ovflowd — APPROVED** [review]
LGTM!

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (eemeli) [reply community-fund/COMMUNITY_FUND_POLICY.md:13]: I get the idea of relying on self-identification for underrepresentation. But I'm still missing some clarity on whether the set of "diverse participants" is exactly equal to the set of people who "identify with an underrepresented group", or does it potentially include some other applicants as well?

If those two sets are the same, why are we using completely different words here and in the request form?

Comment (ovflowd) [reply community-fund/COMMUNITY_FUND_POLICY.md:13]: These sets should be the same. We should apply the sMe wording all places to reduce confusion then

Comment (bensternthal) [reply community-fund/COMMUNITY_FUND_POLICY.md:13]: Yep this is a really good catch. I'll fix the wording and then suggest we merge.

Additional comments/discussion we can spin out into individual issues.

Thanks for the help on this one!

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (bensternthal) [reply community-fund/COMMUNITY_FUND_POLICY.md:72]: Yes we spun out that discussion into a separate issue.

https://github.com/openjs-foundation/cross-project-council/issues/1241

File: community-fund/COMMUNITY_FUND_POLICY.md
Comment (bensternthal) [reply community-fund/COMMUNITY_FUND_POLICY.md:47]: Good call I will sort in a subsequent PR.

**Pull request merged**
